lib/hipcall/resource.rb

```diff
@@ -11,25 +11,25 @@ def initialize(client)
 		def get_request(url, params: {}, headers: {})
 			handle_response client.connection.get(url, params, headers)
 		end
-	
+
 		def post_request(url, body:, headers: {})
 			handle_response client.connection.post(url, body, headers)
 		end
-	
+
 		def patch_request(url, body:, headers: {})
 			handle_response client.connection.patch(url, body, headers)
 		end
-	
+
 		def put_request(url, body:, headers: {})
 			handle_response client.connection.put(url, body, headers)
 		end
-	
+
 		def delete_request(url, params: {}, headers: {})
 			handle_response client.connection.delete(url, params, headers)
 		end
 
 		def handle_response(response)
-			message = response.body["errors"]
+			message = response.body["error"] || response.body["errors"]
```

Comment:
@azimcan Which endpoint do you get `error` and which endpoint do you get `errors`?

Reply:
Example
<img width="935" alt="Screenshot 2024-07-19 at 17 57 51" src="https://github.com/user-attachments/assets/4adb0d79-e902-40d4-9e27-c2459dbb7a64">

Reply:
But this is return `errors:` not `error:` . I don't understand what we should change line 32.

```ruby
message = response.body["errors"]
message = response.body["error"] || response.body["errors"]
````

We can fix the "Internal Server Error" but I don't think we should need to add `response.body["error"] ||` at line 32.